app/services/OvhService.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+class OvhService
```

Comment:
This could actually be named another way, so that it is clearer that it is not only about OVH. However it might be a good idea to create other services for different things (DB modifications, etc)